tests/test_classes.py

```diff
@@ -89,4 +96,15 @@ def test_no_newlines_in_citations():
   test2 = "Bbb\n\n\n2010"
   examples = PapersCited.CitationType([test1, test2])
   examples.cleanup()
-  assert examples.citations == ["Aaa 2007", "Bbb 2010"]
\ No newline at end of file
+  assert examples.citations == ["Aaa 2007", "Bbb 2010"]
+  
+def test_separating_multiple_years():
+  text = "AuthorA (2000; 2002; 2003) thoroughly explored..."
+  matches = PapersCited.get_matches_solo_author(text)
+  matches = PapersCited.CitationType(matches._remove_extra_characters())
+  assert matches._separate_multiple_years() == ["AuthorA 2000", "AuthorA 2002", "AuthorA 2003"]
+  
+  text_abc = "AuthorA (2000a; 2000b; 2000c) thoroughly explored..."
+  matches = PapersCited.get_matches_solo_author(text_abc)
+  matches = PapersCited.CitationType(matches._remove_extra_characters())
+  assert matches._separate_multiple_years() == ["AuthorA 2000a", "AuthorA 2000b", "AuthorA 2000c"]
```

Comment:
General recommendation would be to use one assertion per test method (not a strict rule, but usually you want to have only one reason for a test to fail)

By separating the assertions you can have it visible in the name of the test method what exactly are you testing.

Check commonly used test naming conventions (e.g. "MethodName_StateUnderTest_ExpectedBehavior")